### ISSUE TITLE
NLBPUB to narration - problem med usynlige overskrifter

### ISSUE THREAD

**Issue by espso** [nlbdev/produksjonssystem#275]
Denne pipelinen krasjer hvis den ikke klarer å lage usynlige overskrifter der det trengs. Virker som det skjer mye for tidsskrifter.

Eks. for 624590 (Mat fra Norge)
No usable text content for headline available at: /[1]/[2]/[2]/[2] (section[@id='d6e1507'])

**Comment by josteinaj** [comment]
Ja, jeg tror det er når ting kommer fra Webarch. Så ikke noe som stopper innlesing (siden alt dette skal til TTS uansett), men irriterende å få feilmelding om og om igjen. En fiks vha en XSLT tenker jeg er tingen her, jeg bare har ikke tenkt at dette var så viktig ettersom det ikke hindrer noe sånn direkte, bortsett fra at det er plagsomt.

**Comment by espso** [comment]
Ok skjønner. Siden det uansett ikke skal leses inn er vel ikke det et stort problem. Men det kan jo i teorien skje at vi en gang ønsker at noe fra Webarch leses inn, så kanskje greit å få fikset en gang.
